src/Stratis.Bitcoin/P2P/PeerAddressManagerBehaviour.cs

```diff
@@ -23,6 +24,12 @@ public sealed class PeerAddressManagerBehaviour : NetworkPeerBehavior
         /// <summary>Provider of time functions.</summary>
         private readonly IDateTimeProvider dateTimeProvider;
 
+        /// <summary>It's a logger mate.</summary>
+        private readonly ILogger logger;
```

Comment:
So use it properly. When you introduce logs into previously not logged method, you are supposed to create entry and exit logs.

Reply:
Is that part of the policy?

Reply:
yes